client/src/app/App.js

```diff
@@ -847,7 +847,11 @@ export class App extends PureComponent {
     });
   }
 
-  lintTab = async (tab, contents) => {
+  lintTab = async (tabId, contents) => {
+    const { tabs } = this.state;
+
+    const tab = tabs.find(({ id }) => id === tabId);
```

Comment:
What is this fixing? And if it fixes something where is the test case for it?

Reply:
I found the root cause and fixed it (cf. https://github.com/camunda/camunda-modeler/pull/2399/commits/4c76daaea4d8007987004b0d301639d41d37eeea). No need for this hack.